modules/images-describing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-describing-image-registry-settings-hcp_{context}"]
+= Describing image registry settings for {hcp-title}
+
+
+Description needed !!!!
```

Comment:
This should be commented out or removed before merge, even if the module has been removed from usage. Ideally, we don't really need the module in the PR at all though, right? Could you remove it from the PR?

Reply:
removed it thanks